packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -0,0 +1,174 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+
+import '../../flutter/screen.dart';
+import '../../service_extensions.dart';
+import '../../ui/flutter/label.dart';
+import '../../ui/flutter/service_extension_widgets.dart';
+import '../../ui/flutter/vm_flag_widgets.dart';
+import '../../ui/icons.dart';
+import '../timeline_controller.dart';
+
+class TimelineScreen extends Screen {
+  const TimelineScreen() : super('Timeline');
+
+  @override
+  Widget build(BuildContext context) => const TimelineBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      text: name,
+      icon: Icon(Octicons.getIconData('pulse')),
+    );
+  }
+}
+
+class TimelineBody extends StatefulWidget {
+  const TimelineBody();
+
+  @override
+  TimelineBodyState createState() => TimelineBodyState();
+}
+
+class TimelineBodyState extends State<TimelineBody> {
+  TimelineMode _timelineMode = TimelineMode.frameBased;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Row(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                if (_timelineMode == TimelineMode.frameBased)
+                  OutlineButton(
+                    onPressed: _pauseLiveTimeline,
+                    child: const Label(
+                      FlutterIcons.pause,
+                      'Pause',
+                      minIncludeTextWidth: 900,
+                    ),
+                  ),
+                if (_timelineMode == TimelineMode.frameBased)
+                  OutlineButton(
+                    onPressed: _resumeLiveTimeline,
+                    child: const Label(
+                      FlutterIcons.resume,
+                      'Resume',
+                      minIncludeTextWidth: 900,
+                    ),
+                  ),
+                if (_timelineMode == TimelineMode.full)
```

Comment:
you can spread a list literal to make this code a bit simpler.
```dart
... if (_timelineMode == TimelineMode.full) [
  OutlineButton(bla bla bla),
  OutlineButton(bla bla bla),
],
```

Reply:
done.